src/editor/operations.rs

```diff
@@ -139,21 +139,14 @@ impl Operation {
                 draw_rectangle(cairo, rect, *border, *fill)?;
             }
             Operation::Text {
-                top_left: Point { x, y },
+                top_left,
                 text,
                 colour,
                 font_description,
             } => {
                 info!("Text");
                 cairo.save()?;
-                let layout = pangocairo::create_layout(cairo).unwrap();
-
-                layout.set_markup(text);
-                layout.set_font_description(Some(font_description));
-                cairo.move_to(*x, *y);
-                cairo.set_source_colour(*colour);
-                pangocairo::update_layout(cairo, &layout);
-                pangocairo::show_layout(cairo, &layout);
+                draw_text_centred_at(cairo, top_left, text, colour, font_description)?;
```

Comment:
This doesn't look right

Reply:
Oh, thanks for spotting that, it was left from when I was trying to make the function work..